docs/tool/GUI.ModsManager/FAQ.md

```diff
@@ -0,0 +1,55 @@
+# OpenKH Mod Manager FAQs
+
+### Return to the [Main Page](index.md)
+
+### Special Thanks to:
+* Shananas for the continued upkeep of the OpenKH Mod Manager when something new arises
+* MainMemory for her updating of Panacea to work with the PC releases
+* Xeeynamo for the initial rough version of the program
+* Alios for discovering a major issue with the Mod Manager so that I may include an answer to the problem in this FAQs document
+* Everyone else in the OpenKH community for helping make the modding environment for these games as welcoming and exciting as it has been
+
+## [Table of Contents]()
+* ### [Is extracting my games mandatory for the mod manager to work?](#q-is-extracting-my-games-mandatory-for-the-mod-manager-to-work)
+* ### [Lua mods aren't working!](#q-lua-mods-arent-working)
+* ### [Does OpenKH support Steam Deck/Linux?](#q-does-openkh-support-steam-decklinux)
+* ### [I moved my OpenKH install and now I can't use the mod manager!](#q-i-moved-my-openkh-install-and-now-i-cant-use-the-mod-manager)
+* ### [I can't open the mod manager on my removeable storage device!]()
+* ### [I want to start making mods but don't know how](#q-i-want-to-start-making-mods-but-dont-know-how)
+
+---
+
+### Q: Is extracting my games mandatory for the mod manager to work?
+
+> **A:** For the best user experience, yes. We understand it takes up a lot of storage, but it allows you to retain your unmodified game files through a dynamic file replacement system instead of relying on a slow and tedious patching method. Some tips to reduce the storage used by game extraction are:
+> * Only extract the game(s) you are actively playing/modding.
+> * Enable `Dev View` by going to `Info > Dev View` inside the mod manager. This will allow you to patch your game(s) instead.

```

Comment:
If a user wishes to use mod manager for patching and not for panacea/mod loader they can just not install panacea and the patching menu will be shown not the mod loader menu. this as mentioned in my prevous comment means they need to do any Lua Backend install and configuration manually. the intention of `dev view` is really for someone who made a mod and gets reports that its not working on patching to be able to test without uninstalling panacea (if they wish to troubleshoot the issue.) This is why i kinda hid it so idealy basic users wont see/try enabling it as patching then using panacea seems to cause many issues and anytime the kh2 rando community sees it said person is told to verify their game install remove the backup folder resetup mod manager etc. just too many variables to bother truely troubleshooting and just saying start nearly fresh. (only thing you dont need to do is uninstall and reinstall the game at least thats what i recall kh2 rando help doing)